### PR TITLE
sprint-1 task AIN-04 Login and Register UI

### PR DESCRIPTION
**Pull request by Jeba-Jeba**
**no changes are made to any links on the page.**

Out put on large screen:
UI lg screen

Out put on small/med screen:
UI small view

### CODE REVIEW

**Comment by NatalNineITSolutions** [comment]
Give the screenshots of login response form network tab

**Comment by NatalNineITSolutions** [comment]
Use full width we dont need the whitespace 

image

**Review by NatalNineITSolutions — CHANGES_REQUESTED** [review]
Please give screenshots of login response when you get from network tab 
and we don't need whitespace in login UI i mentioned in the screenshot refer that

File: routes/web.php [hunk 1]
```diff
@@ -54,7 +54,7 @@
 });
 
 
-if ($installed === true) {
+if ($installed === false) {
```
Comment (NatalNineITSolutions) [inline routes/web.php:57]: make this default

**Comment by Jeba-Jeba** [comment]
removed the space on the both side that you told to remove yesterday,
loginnofill

login page after filling details
loginfilled

login response for non  registered user login
wronglogin

Registration UI
regisnofill

registration UI after filling details
regfilled

signUp providing a auth error
signuperr

**Pull request closed without merging**